vignettes/train-and-deploy-with-caret/train-and-deploy-with-caret.Rmd

```diff
@@ -0,0 +1,284 @@
+
+The vehicle year of manufacture does not have a significant effect.
+
+```{r retrieve_model, eval=FALSE}
+download_files_from_run(run, prefix="outputs/")
+accident_model <- readRDS("outputs/model.rds")
+summary(accident_model)
+```
+
+You can use this model to make new predictions:
+
+```{r manual_predict, eval=FALSE}
+newdata <- data.frame( # valid values shown below
+ dvcat="10-24",        # "1-9km/h" "10-24"   "25-39"   "40-54"   "55+"  
+ seatbelt="none",      # "none"   "belted"  
+ frontal="frontal",    # "notfrontal" "frontal"
+ sex="f",              # "f" "m"
+ ageOFocc=16,          # age in years, 16-97
+ yearVeh=2002,         # year of vehicle, 1955-2003
+ airbag="none",        # "none"   "airbag"   
+ occRole="pass"        # "driver" "pass"
+ )
+## predicted probability of death for these variables, as a percentage
+as.numeric(predict(accident_model,newdata, type="response")*100)
+```
+
+## Deploy a prediction endpoint
+
+With your model, you can predict the danger death from of other types of collisions. Use Azure Machine Learning to deploy your model as a prediction service, and then call the model from a Shiny app.
+
+First, register the model you downloaded for deployment:
+
+```{r register_model, eval=FALSE}
+model <- register_model(ws, 
+                        model_path = "outputs/model.rds", 
+                        model_name = "accidents_model",
+                        description = "Predict probablity of auto accident")
+```
+
+A registered model can be any collection of files, but in this case the R model object is sufficient. Azure Machine Learning will track models each time they are deployed, which is our next step.
+
+To create a web service for your model, you first need to create an **entry script**: an R script that will take as input variable values (in JSON format) and output a prediction from your model. Use the file `accident_predict.R`.
+
+Next, define an **environment** for your deployed model. With an environment, you specify R packages (from CRAN or elsewhere) that are needed for your entry script to run. You also provide the values of environment variables that your script can reference to modify its behavior. If you need software other than R to be available, specify a custom Docker image to use. In this tutorial, there are no special requirements, so create an environment with no special attributes:
+
+```{r Create environment, eval=FALSE}
+r_env <- r_environment(name = "basic_env")
+```
+
+Now you have everything you need to create an **inference config**:
+
+``` {r Create inference config, eval=FALSE}
+inference_config <- inference_config(
+  entry_script = "accident_predict.R",
+  source_directory = ".",
+  environment = r_env)
+```
+
+You'll deploy your service to Azure Container Instances. This code provisions a single container to respond to inbound requests, which is suitable for testing and light loads. (For production scale, you can also deploy to Azure Kubernetes Service.)
```

Comment:
Maybe add the link to the reply to AKS example? As a hyperlink

Reply:
will add all the links in my next PR so that Sheri will be aware of the new changes